cpp/iedriver/CommandHandlers/ScreenshotCommandHandler.cpp

```diff
@@ -259,12 +259,18 @@ namespace webdriver {
   }
 
   bool ScreenshotCommandHandler::IsSameColour() {
-    COLORREF firstPixelColour = this->image_->GetPixel(0, 0);
+    BYTE* byteptr = (BYTE*)this->image_->GetBits();
+    int pitch = this->image_->GetPitch();
+    int firstPixelR = *(byteptr);
+    int firstPixelG = *(byteptr + 1);
+    int firstPixelB = *(byteptr + 2);
 
     for (int i = 0; i < this->image_->GetWidth(); i++) {
       for (int j = 0; j < this->image_->GetHeight(); j++) {
-        if (firstPixelColour != this->image_->GetPixel(i, j)) {
-          return false;
+        if (firstPixelR != *(byteptr + pitch*j + 3 * i) ||
```

Comment:
Are you sure there always be 3 bytes per pixel? Looks like there should be 4 bytes in case of image created with `nBPP = 32`:
- https://msdn.microsoft.com/en-us//library/bwea7by5.aspx#CImage::Create
- https://github.com/SeleniumHQ/selenium/blob/selenium-3.3.1/cpp/iedriver/CommandHandlers/ScreenshotCommandHandler.cpp#L245